teuthology/task/ceph_ansible.py

```diff
@@ -253,6 +255,43 @@ def teardown(self):
                 ]
             )
 
+    def collect_logs(self):
+        ctx = self.ctx
+        if ctx.archive is not None and \
+                not (ctx.config.get('archive-on-error') and ctx.summary['success']):
+            # collect logs
+            log.info('Compressing logs...')
+            run.wait(
+                ctx.cluster.run(
+                    args=[
+                        'sudo',
+                        'find',
+                        '/var/log/ceph',
+                        '-name',
+                        '*.log',
+                        '-print0',
+                        run.Raw('|'),
+                        'sudo',
+                        'xargs',
+                        '-0',
+                        '--no-run-if-empty',
+                        '--',
+                        'gzip',
```

Comment:
I see the mode='r|gz' does gzip so i can remove this command but needs to be tested again.